user_docs/en/changes.t2t

```diff
@@ -5,6 +5,17 @@ What's New in NVDA
 %!includeconf: ./locale.t2tconf
 
 = 2023.2 =
+This release most notably introduces the Add-on Store.

```

Comment:
Add-on Store is sometimes written capitalized ("Add-on Store") and sometimes in lower case ("add-on store"). It may be good to standardize in change log and user guide, following the GUI (and hoping that the GUI has not mixed occurrences).

Reply:
I believe this is fixed now